tests/responses/subscriptions_for_userid.json

```diff
@@ -0,0 +1,21 @@
+{
+  "count": 1,
+  "previous": null,
+  "results": [{
+    "status": "active",
+    "deprovision_date": "2016-11-17",
+    "term": "P1Y",
+    "user_id": "fake-user-id-abcdef1234567890abc",
+    "auto_renew": false,
+    "billing_date": "2016-11-02",
+    "properties": {
+      "partner_id": "WL_BOTTOMLINE"
+    },
+    "expiry_date": "2016-11-02",
+    "start_date": "2015-11-02",
+    "sku": "12345678",
+    "type": "wl_subscription_type",
+    "id": "fake-sub-id-abcdef1234567890abcd"
+  }],
+  "next": null
+}
```

Comment:
1 - I don't know how to stub.

If I was going to guess the right way to stub, I would capture the service response and write it somewhere that is easily and regularly be updated - which means storing responses in their own location, not hardcoded into the test

2 - I'm hoping to throw this away

When the required groundwork is finished, I'm hoping to purge all my mocking and stubbing and use integration tests.

> What do you think?

I want to do whatever is easiest to purge AND maintain (just in case this lingers longer than expected), I don't care too much about the aesthetics. I think that is accomplished as-is but not I'm not married to this structure.  If you disagree, that's good enough reason for me to move it.